schema2.proto

```diff
@@ -0,0 +1,218 @@
+syntax = "proto2";
+
+package biscuit.format.schema;
+
+message Biscuit {
+  optional uint32 rootKeyId = 1;
+  required SignedBlock authority = 2;
+  repeated SignedBlock blocks = 3;
+  required Proof proof = 4;
+}
+
+message SignedBlock {
+  required bytes block = 1;
+  required bytes nextKey = 2;
+  required bytes signature = 3;
+}
+
+message Proof {
+  oneof Content {
+    bytes nextSecret = 1;
+    bytes finalSignature = 2;
+  }
+}
+
+message Block {
+  required uint32 index = 1;
+  // ^ is this still useful?
```

Comment:
it seems redundant with the block index in the list. What should be done if there is a discrepancy between explicit indices and the position in the list?

Reply:
the field can be removed, since now the signatures guarantee the order

Reply:
:ok_hand: